### PR TITLE
update local install readme to explain updating reference data

### PR DESCRIPTION
**Pull request by hanars**
Requested in https://github.com/broadinstitute/seqr/discussions/3370#discussioncomment-5909334

### CODE REVIEW

File: deploy/LOCAL_INSTALL.md [hunk 1]
```diff
@@ -85,8 +90,23 @@ This is expected to take a while
...
+   GS_BUCKET=gs://your-bucket       # your google bucket
+   BUILD_VERSION=38                 # can be 37 or 38
    
+   # Update clinvar 
+   gsutil rm -r "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+   gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/clinvar/clinvar.GRCh${BUILD_VERSION}.ht" "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+  
+   # Update all other reference data
+   gsutil rm -r "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
+   gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/all_reference_data/combined_reference_data_grch${BUILD_VERSION}.ht" "${GS_BUCKET}/reference_data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
```
Comment (ShifaSZ) [inline deploy/LOCAL_INSTALL.md:107]: For local installation, the `gsutil rsync` command should sync to a bucket other than `gs://seqr-reference-data`.

File: deploy/LOCAL_INSTALL.md [hunk 2]
```diff
@@ -130,6 +150,20 @@ This is expected to take a while
...
+   ```bash
+   BUILD_VERSION=38                 # can be 37 or 38
+   
+   # Update clinvar 
+   docker-compose exec pipeline-runner rm -rf "/seqr-reference-data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+   docker-compose exec pipeline-runner gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/clinvar/clinvar.GRCh${BUILD_VERSION}.ht" "/seqr-reference-data/GRCh${BUILD_VERSION}/clinvar.GRCh${BUILD_VERSION}.ht"
+  
+   # Update all other reference data
+   docker-compose exec pipeline-runner rm -rf "/seqr-reference-data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
+   docker-compose exec pipeline-runner gsutil rsync -r "gs://seqr-reference-data/GRCh${BUILD_VERSION}/all_reference_data/combined_reference_data_grch${BUILD_VERSION}.ht" "/seqr-reference-data/GRCh${BUILD_VERSION}/combined_reference_data_grch${BUILD_VERSION}.ht"
```
Comment (ShifaSZ) [inline deploy/LOCAL_INSTALL.md:165]: No `gsutil rsync` commands are required for the on-prem option.

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: deploy/LOCAL_INSTALL.md (hunk 1 above)
Comment (hanars) [reply deploy/LOCAL_INSTALL.md:107]: But thats where they get the data from. Its rsync'ing the content of that public bucket to the local machine which is exactly whats done in the helper scripts we tell them to run above

File: deploy/LOCAL_INSTALL.md (hunk 2 above)
Comment (hanars) [reply deploy/LOCAL_INSTALL.md:165]: They need to rsync the content of our reference data bucket on GCP to their local machine

File: deploy/LOCAL_INSTALL.md (hunk 1 above)
Comment (ShifaSZ) [reply deploy/LOCAL_INSTALL.md:107]: I see. The `gs://seqr-reference-data` is accessible to the public.

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**